activities/philanthropic-fundraising/ANBI.md

```diff
@@ -0,0 +1,27 @@
+---
+type: Guide
+---
+
+# How to donate in the Netherlands to a Public Benefit Organization (ANBI)
+
+You may deduct a donation to an ANBI (public benefit organization) in your income tax return. However, the conditions are different for a periodic gift and an ordinairy gift.
+
+## Periodic donation
+
+Your donation is a periodic donation if it meets the following 3 conditions:
+
```

Comment:
- does it need to meet 1 of the following 3 conditions, or all of them? 
- use numbers for sequential steps. If the list is unordered, use bullets.

Reply:
Ditto for the list below.